### PR TITLE
bcm2711_bootloader_config.md: improve XHCI_DEBUG section.

### PR DESCRIPTION
**Pull request by andrum99**
- Since the parameter must be specified in hex, it makes more sense to list the contents of the bitfield in hex too.
- Standardise on `debug` (remove one instance of `trace`).
- Simplify things a bit.
- Add basic instructions for how to combine values - feel free to remove the bit in brackets - not sure about that part.

### CODE REVIEW

File: hardware/raspberrypi/bcm2711_bootloader_config.md [hunk 1]
```diff
@@ -277,24 +277,28 @@ Default: 1000 (1 second)
...
+
+(`0x` means these numbers are in hexadecimal. If you're not sure how to add these up, simply type the sum into Google, e.g. `=0x2+0x4`).
 
 ```
 # Example: Enable mass storage and USB descriptor logging
 XHCI_DEBUG=0x3
 ```
 
-Default: 0x0  
+Default: 0x0 (no USB debug messages enabled)
```
Comment (lurch) [inline hardware/raspberrypi/bcm2711_bootloader_config.md:301]: You need to keep the two space characters at the end of the line, to force `Default:` and `Version:` to be displayed on separate lines.

**Comment by lurch** [comment]
ping @timg236

File: hardware/raspberrypi/bcm2711_bootloader_config.md (hunk 1 above)
Comment (andrum99) [reply hardware/raspberrypi/bcm2711_bootloader_config.md]: Thanks - I hadn't realised they were intentional. I now know how to do line breaks in markdown 👍

**Comment by JamesH65** [comment]
Thanks!

**Pull request merged**